components/automate-ui/src/app/pages/notifications/notifications.component.scss

```diff
@@ -31,6 +31,16 @@
     margin-right: 1em;
   }
 
+  chef-td {
+    display: flex;
+    flex-direction: row;
+
+    a {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+
```

Comment:
This might end up being a stop gap until I swap out the tables, however this table content is unique in that it has two items inside its table-cell, which requires a little bit different of CSS.